packages/controller/test/lib/testObjects.ts

```diff
@@ -99,7 +99,7 @@ export function register(it: Mocha.TestFunction, expect: Chai.ExpectStatic, cont
                     expect(idOrName).to.be.equal('test3');
                     expect(id).to.be.equal(undefined);
 
-                    objects.findObject('test2', 'channel', (err, id, idOrName) => {
+                    objects.findObject('test2', 'boolean', (err, id, idOrName) => {
```

Comment:
was there a need to change the test rather then adding new cases?

Reply:
Because earlier type could be any string. Now it is narrowed to only state types and it does not have 'channel' in it